content/ui-ux.md

```diff
@@ -14,11 +14,40 @@ After reading this guide, you'll know:
 5. How to build components that can cope with a variety of different data sources.
 6. How to use animation to keep users informed of changes.
 
-<h2 id="components">UI components</h2>
+<h2 id="view-layers">View layers</h2>
+
+Meteor officially supports three user interface (UI) rendering libraries, [Blaze](blaze.html), [React](react.html) and [Angular](http://www.angular-meteor.com). Blaze was created as part of Meteor when it launched in 2011, React was created by Facebook in 2013, and Angular was created by Google in 2010. All three have been used successfully by large production apps. Blaze is the easiest to learn and has the most full-stack Meteor packages, but React and Angular are more developed and have larger communities.
+
+
+**Syntax**
+- Blaze uses an easy-to-learn [Handlebars](http://handlebarsjs.com)-like template syntax, with logic like `{% raw %}{{#if}}{% endraw %}` and `{% raw %}{{#each}}{% endraw %}` interspersed in your HTML files. Template functions and CSS-selector events maps are written in Javascript files.
+- React uses [JSX](https://facebook.github.io/react/docs/jsx-in-depth.html), with which you write your HTML in Javascript. While it doesn't have the logic-view separation most libraries have, it also has the most flexibility. Template functions are TODO
```

Comment:
OMG I've been spelling it wrong!! At least it's not actually the language
I've been using for the past five years... OMG it's "CoffeeScript" too?!
😳😝

On Mon, May 2, 2016 at 1:33 PM, Tom Coleman <notifications@github.com>
wrote:

> In content/ui-ux.md
> <https://github.com/meteor/guide/pull/406#discussion_r61771936>:
>
> > @@ -14,11 +14,40 @@ After reading this guide, you'll know:
> >  5. How to build components that can cope with a variety of different data sources.
> >  6. How to use animation to keep users informed of changes.
> >
> > -<h2 id="components">UI components</h2>
> > +<h2 id="view-layers">View layers</h2>
> > +
> > +Meteor officially supports three user interface (UI) rendering libraries, [Blaze](blaze.html), [React](react.html) and [Angular](http://www.angular-meteor.com). Blaze was created as part of Meteor when it launched in 2011, React was created by Facebook in 2013, and Angular was created by Google in 2010. All three have been used successfully by large production apps. Blaze is the easiest to learn and has the most full-stack Meteor packages, but React and Angular are more developed and have larger communities.
> > +
> > +
> > +**Syntax**
> > +- Blaze uses an easy-to-learn [Handlebars](http://handlebarsjs.com)-like template syntax, with logic like `{% raw %}{{#if}}{% endraw %}` and `{% raw %}{{#each}}{% endraw %}` interspersed in your HTML files. Template functions and CSS-selector events maps are written in Javascript files.
> > +- React uses [JSX](https://facebook.github.io/react/docs/jsx-in-depth.html), with which you write your HTML in Javascript. While it doesn't have the logic-view separation most libraries have, it also has the most flexibility. Template functions are TODO
>
> Javascript -> JavaScript.
>
> "Template functions and event handlers are defined in the same file as the
> HTML part of the component, which usually makes it easier to understand how
> they are tied together" - maybe?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/meteor/guide/pull/406/files/de055e6ec05bc51f4efd90b66f96d86e421635de#r61771936>
>